packages/graphql/tests/schema/directives/selectable.test.ts

```diff
@@ -0,0 +1,717 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { printSchemaWithDirectives } from "@graphql-tools/utils";
+import { lexicographicSortSchema } from "graphql/utilities";
+import { gql } from "graphql-tag";
+import { Neo4jGraphQL } from "../../../src";
+import { TestSubscriptionsPlugin } from "../../utils/TestSubscriptionPlugin";
+
+describe("@selectable", () => {
+    let subscriptionPlugin: TestSubscriptionsPlugin;
+
+    beforeAll(() => {
+        subscriptionPlugin = new TestSubscriptionsPlugin();
+    });
+
+    test("Disable read fields", async () => {
+        const typeDefs = gql`
+            type Movie {
```

Comment:
Is it worth explicitly enabling subscriptions/aggregations for these tests in preparation for the 4.0.0 changes?
```suggestion
            type Movie @query(aggregate: true) {
```